src/OrchardCore.Themes/TheBlogTheme/Views/Content-Blog.liquid

```diff
@@ -1,19 +1,24 @@
 {% zone "Header" %}
-<!-- Page Header -->
-<!-- Set your background image for this header on the line below. -->
-<header class="masthead" style="background-image: url('{{ Model.ContentItem.Content.Blog.Image.Paths.first | asset_url }}')">
-    <div class="overlay"></div>
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-8 col-md-10 mx-auto">
-                <div class="site-heading">
-                    <h1>{{ Model.ContentItem.DisplayText }}</h1>
-                    <span class="subheading">{{ Model.Content.HtmlBodyPart | shape_render }}</span>
+    <!-- Page Header -->
+    <!-- Set your background image for this header on the line below. -->
+    {% assign imagePath = Model.ContentItem.Content.Blog.Image.Paths.first %}
+    {% if imagePath == empty %}
```

Comment:
Don't use `== empty`, use `== nil` or it won't work.
I just tested on an existing blog.

Reply:
Ok